qless/jobs.py

```diff
@@ -0,0 +1,49 @@
+'''Generic utility jobs.'''
+
+import subprocess
+
+from . import logger
+
+
+class SubprocessJob(object):
```

Comment:
I'm not sure I would make this directly available in qless-py. I think it makes sense for our described use case, and if someone wants to do the same thing, it's not that hard for them to reproduce this work, but I wouldn't want to open every qless install up to this.